### PR TITLE
include rubygems how-to installation

### PR DESCRIPTION
**Pull request by maurcarvalho**
@igorescobar this PR is related to https://github.com/igorescobar/jQuery-Mask-Plugin/issues/477
Thank you,
MC

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,26 @@ https://cdnjs.com/libraries/jquery.mask
...
 
+## RubyGems
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'jquery_mask_rails', '~> 0.1.0'
+```
+
+And then execute:
```
Comment (igorescobar) [inline README.md]: I don't think that all those instructions are necessary. If they want to use they can locate the gem's owner and then find further instructions.

Comment (maurcarvalho) [inline README.md]: 👍

Comment (maurcarvalho) [inline README.md]: Changing it ..

**Comment by maurcarvalho** [comment]
done 👍

File: README.md (hunk 1 above)
Comment (maurcarvalho) [inline README.md]: done

File: README.md [hunk 2]
```diff
@@ -44,6 +44,11 @@ https://cdnjs.com/libraries/jquery.mask
 ### JSDelivr
 http://www.jsdelivr.com/projects/jquery.mask
 
+## RubyGems
+```ruby
+gem 'jquery_mask_rails', '~> 0.1.0'
```
Comment (igorescobar) [inline README.md]: @maurcarvalho Would be better if remove the version. As you upgrade the gem this would be outdated.

Comment (maurcarvalho) [inline README.md]: Indeed. I updated it.
Thank you

**Comment by igorescobar** [comment]
thanks @maurcarvalho !

File: README.md (hunk 1 above)
Comment (jicardu) [inline README.md]: I needed to write `//= require jquery.mask` in the application.js to make it work

Comment (maurcarvalho) [inline README.md]: @igorescobar do you want to reconsider?

**Pull request merged**